src/js/components/FrameworkConfigurationForm.js

```diff
@@ -245,11 +247,26 @@ class FrameworkConfigurationForm extends Component {
     } = this.props;
 
     const TitleField = props => {
-      return (
-        <h2 className="flush-top short-bottom">
-          {StringUtil.capitalizeEveryWord(props.title)}
-        </h2>
-      );
+      const level = props.formContext.level;
+      if (level === FrameworkConfigurationConstants.headingLevel.H1) {
+        return (
+          <h1 className="flush-top short-bottom">
+            {StringUtil.capitalizeEveryWord(props.title)}
+          </h1>
+        );
+      } else if (level === FrameworkConfigurationConstants.headingLevel.H2) {
```

Comment:
Personal preference note.
Since we are returning we could maybe remove the else if and add the last statement as the default return.

Reply:
Also, now that the tests are working we can probably remove the "pass" label.

Reply:
Flattened this, thanks